fleet-examples/fleets/elemental/elemental-operator/fleet.yaml

```diff
@@ -0,0 +1,7 @@
+defaultNamespace: cattle-elemental-system
+helm:
+  releaseName: elemental-operator
+  # I've tried with the OCI one and it breaks Fleet https://github.com/rancher/fleet/issues/1606
```

Comment:
Just FYI, there's now a working example in the mentioned issue where we specify the version. Not sure if we want to change it here.